src/Compilers/CSharp/Test/Syntax/Syntax/SyntaxNormalizerTests.cs

```diff
@@ -16,6 +16,87 @@ namespace Microsoft.CodeAnalysis.CSharp.UnitTests
 {
     public class SyntaxNormalizerTests
     {
+        [Fact, WorkItem(52543, "https://github.com/dotnet/roslyn/issues/52543")]
+        public void TestNormalizePatternInIf()
+        {
+            TestNormalizeStatement(
+                @"{object x = 1;
+                if (x is {})
+                {
+                }
+                if (x is {} t)
+                {
+                }
+                if (x is int {} t2)
+                {
+                }
+                if (x is System.ValueTuple<int, int>(_, _) { Item1: > 10 } t3)
+                {
+                }
+                if (x is System.ValueTuple<int, int>(_, _) { Item1: > 10, Item2: < 20 })
+                {
+                }
+}",
+                "{\r\n  object x = 1;\r\n  if (x is { })\r\n  {\r\n  }\r\n\r\n  if (x is { } t)\r\n  {\r\n  }\r\n\r\n  if (x is int { } t2)\r\n  {\r\n  }\r\n\r\n  if (x is System.ValueTuple<int, int> (_, _) { Item1: > 10 } t3)\r\n  {\r\n  }\r\n\r\n  if (x is System.ValueTuple<int, int> (_, _) { Item1: > 10, Item2: < 20 })\r\n  {\r\n  }\r\n}"
```

Comment:
this is really hard to read.  can you please just put in normal newlines?

Reply:
An ubuntu build issued an error - that was the reason...
But I do not like it, too.

Reply:
Yes, on a (ubuntu) machine it failed because of "\r\n" vs. "\n"...

Reply:
But I reverted that